docs/release.md

```diff
@@ -324,9 +324,39 @@ sourced from the delta of PRs merged since the last official release. The text
 from those PRs are sourced directly from the ```release-notes``` section in
 each PRs description.
 
-Below is an example of getting the release notes for v0.31.0-rc.0
+## Using GitHub labels for Release Notes
+
+We use GitHub labels to filter release notes, first using a 'kind/' label and
+second using a 'sig/' label where applicable.
+In some cases, 'kubevirt-bot' applies one or more of these labels automatically,
+however they might need to be manually applied by either the author or a reviewer of the PR.
+
+If a PR matches more than one 'kind/' or 'sig/' label, add all that apply. The release note will be included in whichever takes the greater priority.
+
+As per the PR template, if the PR requires additional action from users switching to the new release, include the string "action required" in the release note. The PR will be flagged accordingly.
```

Comment:
Is the PR description template going to change to explain this?

Reply:
Makes sense :) I've added it to the release note part of the template

Reply:
Awesome! Thank you :)

Will the bot add these labels if they appear in the PR description, or only if they appear in comments?
Also, if someone makes a mistake, e.g. uses `/sig compute` instead of `/sig network`, then changes the PR description, would the bot change the label automatically?

Perhaps @dhiller can help us answer these questions

Reply:
Also: is there a way to remove these labels after they're added?

Reply:
@dhiller ping :)

Reply:
To answer the last question: adding and removing labels manually can be done as described here: https://prow.ci.kubevirt.io/command-help#kind

For the rest of the questions, I first need to dig into what this PR is about again.

Reply:
According whether it works inside or outside a release-notes block: looking at the [regexes here](https://github.com/kubernetes/test-infra/blob/eb520859a75d5f802f3fccf604aac554b3de05e0/prow/plugins/label/label.go#L42) the only requirement I see is that each of the label commands needs to be at a new line with no spaces allowed before it.

Reply:
Thanks for the clarifications!

Reply:
Thus I think the line 336 needs a change to express it.

@aburdenthehand @iholder101

Reply:
To the last question, IIRC it works both on the PR description and on comments.